.github/workflows/packaging-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - 'ubuntu/**'
+
+concurrency:
+  group: 'ci-${{ github.workflow }}-${{ github.ref }}'
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: sh -ex {0}
+
+env:
+  RELEASE: focal
+
+jobs:
+  daily-ppa-recipe-check:
```

Comment:
While this isn't necessarily incorrect, the daily PPA isn't the only thing to benefit from this CI job. How about something more generic, like `check-patches`? Also, a `name` key would produce a more user-friendly output, such as `name: Check Patches`